backend/biological_models_app/__init__.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask
+from flask_cors import CORS #For connection between flask and Vue
+import mysql.connector #For connecting database
+
+app = Flask(__name__) #Define app within init -> used throughout program
+
+CORS(app, resources={r'/*':{'origins':"*"}})
+
+#Connect to MySQL and create cursor to make queries
+db = mysql.connector.connect(host="localhost", user="root", database="biologicalmodelswebapp", password="030103")
+
+#Import Blueprints here to avoid circular imports
+from biological_models_app.user.routes import user
+from biological_models_app.pred_prey.routes import pred_prey
+from biological_models_app.competing_species.routes import competing_species
+from biological_models_app.SIR.routes import SIR
+from biological_models_app.SEIDR.routes import SEIDR
```

Comment:
Move to top of file with other imports

Reply:
They need to go towards the bottom to avoid circular imports, as I import app into all of those modules